fairlearn/datasets/_fetch_boston.py

```diff
@@ -50,7 +50,7 @@ def fetch_boston(*, cache=True, data_home=None,
     RAD      index of accessibility to radial highways
     TAX      full-value property-tax rate per $10,000
     PTRATIO  pupil-teacher ratio by town
-    B        1000(Bk - 0.63)^2 where Bk is the proportion of blacks by town
+    B        1000(Bk - 0.63)^2 where Bk is the proportion of black people by town
```

Comment:
Can we capitalize Black? I forget where I read about this but the first thing that showed up in my search was https://apastyle.apa.org/style-grammar-guidelines/bias-free-language/racial-ethnic-minorities

```suggestion
    B        1000(Bk - 0.63)^2 where Bk is the proportion of Black people by town
```

Reply:
I'll commit this since you gave "thumbs up" 🙂